### PR TITLE
Bugfix/ad 55 onsubmit incorrect handling

### PR DESCRIPTION
**Pull request by er1z**

### CODE REVIEW

File: src/Resources/public/css/adyen.css [hunk 1]
```diff
@@ -29,7 +29,9 @@ body .dropin-container .adyen-checkout__threeds2__challenge iframe
     margin: 1.5rem 0;
 }
 
-.adyen-submit.hidden
+.adyen-submit.hidden,
+.dropin-container.hidden,
+.adyen-method-grid.hidden
 {
     display: none;
 }
```
Comment (Harvel218) [inline src/Resources/public/css/adyen.css]: missing new line at the end, wrong code format in almost whole file.
use this https://www.freeformatter.com/css-beautifier.html
to format css

File: src/Resources/public/js/checkout.js [hunk 2]
```diff
@@ -34,14 +34,14 @@ document.addEventListener('DOMContentLoaded', (e) => {
```
Comment (Harvel218) [inline src/Resources/public/js/checkout.js:34]: Rename the variables (remove the dollar sign, e.g. '$') - this is typical for jquery, and you wrote the vanilla js code, don't confuse the reader! : D

File: src/Resources/public/js/checkout.js [hunk 3]
```diff
@@ -34,14 +34,14 @@ document.addEventListener('DOMContentLoaded', (e) => {
 
         showSubmitButton(false);
 
-        adyenMethod.querySelector('.adyen-method-grid, .dropin-container').style.display = '';
+        adyenMethod.querySelector('.adyen-method-grid, .dropin-container').classList.remove('hidden');
```
Comment (Harvel218) [inline src/Resources/public/js/checkout.js:37]: this works but it i dont think tis correct.

here querySelector will find First element with matching class - the rest will be missed - i dont know what you want to do here, but if you want just 1 specific element, you shouldnt need 2 classes to find it.

If you want ALL elements with these classes, you should use querySelectorAll, and then in array.forEach((item)=>{
item.classList.remove('hidden')
})

File: src/Resources/public/js/checkout.js (hunk 2 above)
Comment (Harvel218) [inline src/Resources/public/js/checkout.js:34]: Line 19: showSubmitButton - schould be toggleSubmitButton or handleSubmitButton - cuz its toggling it's visibility, not just showing :D

File: src/Resources/public/js/dropin.js [hunk 4]
```diff
@@ -10,6 +10,7 @@
 
         let checkout = null;
         let configuration = {};
+        let $form = $container.closest('form');
```
Comment (Harvel218) [inline src/Resources/public/js/dropin.js:13]: change variable name (dolar sign thing as above)

File: src/Resources/public/js/dropin.js [hunk 5]
```diff
@@ -76,6 +101,8 @@
         };
 
         const init = () => {
```
Comment (Harvel218) [inline src/Resources/public/js/dropin.js:103]: Inside init function you can add if statements, to check if its possible to run functions, and return nothing/error if not - you will not initialize adyen then - but idk if its smthing you need

**Review by Harvel218 — CHANGES_REQUESTED** [review]

File: src/Resources/public/js/checkout.js (hunk 3 above)
Comment (er1z) [reply src/Resources/public/js/checkout.js:37]: It's intentional — the same code is executed in two different contexts; first `.adyen-method-grid`, second `.dropin-container`, it won't happen for two at once.

File: src/Resources/public/js/dropin.js (hunk 5 above)
Comment (er1z) [reply src/Resources/public/js/dropin.js:103]: What do you recommend?

File: src/Resources/public/js/checkout.js (hunk 2 above)
Comment (er1z) [reply src/Resources/public/js/checkout.js:34]: Intentional, we've discussed via DM.

File: src/Resources/public/css/adyen.css (hunk 1 above)
Comment (er1z) [reply src/Resources/public/css/adyen.css]: phpstorm re-format will be sufficient?

**Review by Harvel218 — APPROVED** [review]

**Pull request merged**